packages/ui/src/hooks/useDevice.ts

```diff
@@ -0,0 +1,27 @@
+import { useMediaQuery } from '@react-hookz/web';
+
+const BREAKPOINT_SM = 640;
+const BREAKPOINT_MD = 768;
+const BREAKPOINT_LG = 1024;
+const BREAKPOINT_XL = 1280;
+const BREAKPOINT_2XL = 1536;
```

Comment:
I saw you were importing colors from Tailwind in a Storybook doc, can't you do the same for breakpoints? That would make the maintenance much easier, as you won't have to keep these values in sync manually.

Reply:
This required importing the `tailwind.config.js` and resolving with `tailwindcss/resolveConfig`, it brought up issues around `module.exports` used in the tailwind.config file and changing to export default caused tailwind to use default config (not able to use custom config)

Reply:
Okay, thanks for checking this out 👍🏻 
Let's just remember to keep those values in sync. Maybe worth adding a comment in `tailwind.config.js` that values in `useDevices` need to be kept in sync with the config.

Reply:
https://github.com/prezly/theme-kit-js/pull/423/commits/636e7d419d73a41752cf4e1e134276da2532a013